sufia.gemspec

```diff
@@ -23,8 +23,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'hydra-head', '>= 10.4.0'
   spec.add_dependency 'hydra-batch-edit', '~> 2.0'
   spec.add_dependency 'browse-everything', '>= 0.10.3'
-  # Workaround for https://github.com/projecthydra-labs/hyrax/issues/546
-  spec.add_dependency 'blacklight', '~> 6.6', '< 6.8.0'
+  spec.add_dependency 'blacklight', '~> 6.6', '< 7'
```

Comment:
I wonder if the `'< 7' is adding anything here. Shouldn't the `'~> 6.6'` twiddly-wakka be sufficient to keep the version less than 7?

Reply:
Yes, you're totally right; holdover from when I was messing around. Thanks for noticing.